src/QueryExecutionError.php

```diff
@@ -11,7 +11,7 @@ class QueryExecutionError extends QueryError
 
     public function __construct(string $message, array $diagnostics, \Throwable $previous = null)
     {
-        parent::__construct($message, 0, $previous);
+        parent::__construct($message, $diagnostics['sql'], $previous);
```

Comment:
A string should not be used for the code.

Reply:
Why? parent constructor have exactly string parameter at this place? Any ideas how to pass this without string?

Reply:
So it does, I didn't realize `QueryError` had a custom constructor taking the SQL query. Not providing it to `QueryExecutionError` was certainly an oversight before.

Reply:
What should I do?